### PR TITLE
[内容更新] src/docs/development/ui/animations/staggered-animations.md

### LINKED ISSUE #786: '交织动画' page issue
Page URL: https://flutter.cn/docs/development/ui/animations/staggered-animations.html
Page source: https://github.com/cfug/flutter.cn/tree/master/src/docs/development/ui/animations/staggered-animations.md

Found a typo? You can fix it yourself by going to the page source and clicking the pencil icon. Or finish creating this issue.

Description of issue: 
## Prob 1
under #Resources#, there are lots of unknown links

## Prob 2
https://github.com/cfug/flutter.cn/blob/master/src/docs/development/ui/animations/staggered-animations.md#%E4%B8%80%E4%B8%AA%E4%BA%A4%E7%BB%87%E5%8A%A8%E7%94%BB%E7%9A%84%E5%9F%BA%E7%A1%80%E7%BB%93%E6%9E%84

这里列表包括...前后的空格用错了吧，并且这里翻译很奇怪

## Prob 3
文中有大量的引用括弧没有被落实，如：

> 例子
> 本指南解释了 basic_staggered_animation 示例。你也可参考更复杂的例子，staggered_pic_selection。
> 
> basic_staggered_animation
> 展示一个单独的 widget 的一系列连续和重叠动画。轻触屏幕开始一个动画，改变不透明度，大小，形状、颜色和填充。
> 
> staggered_pic_selection
> 展示从一个以三种大小显示的图像列表中删除一个图像。这个例子使用两个 animation controllers: 一个用于控制图像的选择/取消> 选择, 一个用于控制删除图像. 选择/取消选择的动画是交织动画。 (想看到这种效果，你可能需要增加 timeDilation 的数值。) 选择> 最大的一个图像，它会收缩同时在一个在蓝色圆圈里显示一个对勾，然后，选择一个最小的图像，最大的图像会展开同时对勾消失。在最大的图像结束展开前，最小的图像会收缩并显示对勾。这个交织行为比较类似于你在 Google Photos 中看到的效果。
> 
> The following video demonstrates the animation performed by basic_staggered_animation:

### PR DESCRIPTION
**Pull request by MeandNi**
fix #786

### CODE REVIEW

**Comment by PassionPenguin** [comment]
i see, but would you like to change like this:

> 指南解释了 basic_staggered_animation 示例。你也可参考更复杂的例子【：】staggered_pic_selection。

=&gt; 此处将逗号改为句号（汉语中少见逗号分隔例子吧，读起来有点难受，虽然原文是如此分割的）

> basic_staggered_animation
> Shows a series of sequential and overlapping animations of a single widget. Tapping the screen begins an animation that changes opacity, size, shape, color, and padding.

> basic_staggered_animation
> 展示一个单独的 widget 的一系列连续和重叠动画。 轻触屏幕开始一个动画，改变不透明度，大小，形状、颜色和填充。

=&gt; 记得给链接

> staggered_pic_selection
> Shows deleting an image from a list of images displayed in one of three sizes. This example uses two animation controllers: one for image selection/deselection, and one for image deletion. The selection/deselection animation is staggered. (To see this effect, you might need to increase the timeDilation value.) Select one of the largest images—it shrinks as it displays a checkmark inside a blue circle. Next, select one of the smallest images—the large image expands as the checkmark disappears. Before the large image has finished expanding, the small image shrinks to display its checkmark. This staggered behavior is similar to what you might see in Google Photos.

> staggered_pic_selection
> 展示从一个以三种大小显示的图像列表中删除一个图像。 这个例子使用两个 animation controllers: 一个用于控制图像的选择/取消选择, 一个用于控制删除图像. 选择/取消选择的动画是交织动画。 (想看到这种效果，你可能需要增加 timeDilation 的数值。) 选择最大的一个图像，它会收缩同时在一个在蓝色圆圈里显示一个对勾， 然后，选择一个最小的图像，最大的图像会展开同时对勾消失。 在最大的图像结束展开前，最小的图像会收缩并显示对勾。 这个交织行为比较类似于你在 Google Photos 中看到的效果。 {{site.alert.end}}

=&gt; 记得给链接

- by 苏苏的 [PassionPenguin](https://github.com/PassionPenguin)

File: src/docs/development/ui/animations/staggered-animations.md [hunk 1]
```diff
@@ -179,21 +181,21 @@ After running forward, the animation runs in reverse.
   * Regardless of how long the animation lasts in real time,
     the controller's values must be between 0.0 and 1.0, inclusive.
     
-    无论动画在真实时间中播放多长时间，控制器的值必须在 0.0　和 1.0 之间， 包括 0.0　和 1.0。
+    无论动画在真实时间中播放多长时间，控制器的值必须在 0.0 和 1.0 之间， 包括 0.0 和 1.0。
```
Comment (AlexV525) [inline src/docs/development/ui/animations/staggered-animations.md:184]:
```suggestion
    无论动画在真实时间中播放多长时间，控制器的值必须在 0.0 和 1.0 之间，包括 0.0 和 1.0。
```

**Comment by MeandNi** [comment]
> i see, but would you like to change like this:
> 
> > 指南解释了 basic_staggered_animation 示例。你也可参考更复杂的例子【：】staggered_pic_selection。
> 
> => 此处将逗号改为句号（汉语中少见逗号分隔例子吧，读起来有点难受，虽然原文是如此分割的）
> 
> => 是否可以加个链接，不然总感觉像是英语没翻译干净（
> 
> * by 苏苏的 [PassionPenguin](https://github.com/PassionPenguin)

fixed。感谢反馈，链接应该还是要尊重一下原文，下文有列举出。

File: src/docs/development/ui/animations/staggered-animations.md [hunk 2]
```diff
@@ -98,7 +98,7 @@ This guide shows how to build a staggered animation in Flutter.
     
   [staggered_pic_selection][]
   <br> 展示从一个以三种大小显示的图像列表中删除一个图像。
-    这个例子使用两个 [animation controllers][]:
+    这个例子使用两个 [animation controllers][]：
```
Comment (AlexV525) [inline src/docs/development/ui/animations/staggered-animations.md:101]:
```suggestion
    这个例子使用了两个 [动画控制器][animation controllers]：
```

**Review by AlexV525 — APPROVED** [review]
image

**Pull request merged**